module/VuFind/src/VuFind/View/Helper/Root/Component.php

```diff
@@ -42,6 +42,13 @@
  */
 class Component extends AbstractHelper
 {
+    /**
+     * Local cache
+     *
+     * @var array
+     */
+    protected array $cache = [];
```

Comment:
A little more specificity here might be helpful:

```suggestion
    /**
     * Local cache of CSS class names for components (component name => class name).
     *
     * @var array
     */
    protected array $cssClassCache = [];
```

Reply:
...and obviously if you agree to change the name of the property, you'll have to adjust other references accordingly. :-)

Reply:
sure, old habit naming with shorter names, descriptive names are nice.